src/helpers/attributes.ts

```diff
@@ -43,16 +43,30 @@ export const tagConverter = (value: string | null): Tag => {
  * </oe-spectrogram>
  * ```
  */
-export const callbackConverter = (value: string | ((...params: any) => any)) => {
+export const callbackConverter = <T>(value: string | null | ((...params: any[]) => T)) => {
```

Comment:
not keen on the existence of this. I assume this is here so we can convert string attributes to functions... maybe we just don't allow that, even if it does look nice.

not requiring this to be changed

Reply:
Yeah, this is so that you can write javascript directly in html attributes

e.g.

```html
<oe-tag-prompt
	search="() => [{ text: 'test' }]"
></oe-tag-prompt>
```

Reply:
yeah this is a very weak dislike. leave it as is... just smells and I'm not sure why